dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -372,6 +372,18 @@ void BlobStore::read(PageIDAndEntriesV3 & entries, const PageHandler & handler,
     for (const auto & p : entries)
         buf_size = std::max(buf_size, p.second.size);
 
+    if (buf_size == 0)
+    {
```

Comment:
Add some comments to explain this early exit for buf_size == 0 is to avoid calling malloc with size 0